flow/designs/sky130hd/chameleon_hier/DFFRAM_4K/pin_order.cfg

```diff
@@ -0,0 +1,8 @@
+#N
+Do.*
+A.*
+CLK
+WE.*
+EN
+Di.*
+
```

Comment:
What are the pin_order.cfg files used for? They don't appear to be used anywhere.

Reply:
I've removed it